freedomofpress/securedrop-workstation-keyring
# Changelog, rpm-build, and qubes-builder

Add changelog, rpm-build, and qubes-builder and qubes-builder compatibility. (Some commits pulled in from qubes-skeleton repo).

The reproducible rpm logic from our own build process is retained with the containerized build script (`make build-rpm`), but will not be retained via qubes-builder - needs discussion. We also deviate slightly from the qubes-skeleton repo (setting a version, release, and changelog in the spec vs via commandline parameters like `@CHANGELOG@`).

To test rpm reproducibility:

`make build-rpm` from the tip of this branch and see 

```
Build complete! RPMs and their checksums are:

108c022a5955ccbb0c9c17b9eec1e395c36f14d9ba0babb47679f1076f9540cf  rpm-build/RPMS/noarch/securedrop-workstation-keyring-0.1.0-1.fc37.noarch.rpm
2e4fecfa97e9e5708a0e18b8eb1a0d536b9251097816bf60b66b016cd555471a  rpm-build/RPMS/noarch/securedrop-workstation-keyring-0.1.0-1.noarch.rpm

```
